spec/factories/seeds/participant_profile_factory.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory(:seed_participant_profile, class: "ParticipantProfile") do
+    factory(:seed_ecf_participant_profile, class: "ParticipantProfile::ECF") do
+      type { "ParticipantProfile::ECF" }
+    end
+
+    schedule { Finance::Schedule.first }
+
+    trait :with_user do
+      user { build(:seed_user) }
```

Comment:
I think a profile has a user through teacher profile 🤔

Reply:
Yeah good call, afraid I copied and pasted a few times and that looks like it was meant for `participant_identity` 🤦🏽‍♂️